tensor/stats/stats/stats.go

```diff
@@ -4,70 +4,133 @@
 
 package stats
 
+import (
+	"strings"
+
+	"cogentcore.org/core/base/errors"
+	"cogentcore.org/core/tensor"
+)
+
 //go:generate core generate
 
+func init() {
+	tensor.AddFunc(Count.FuncName(), CountFunc, 1)
+	tensor.AddFunc(Sum.FuncName(), SumFunc, 1)
+	tensor.AddFunc(SumAbs.FuncName(), SumAbsFunc, 1)
+	tensor.AddFunc(L1Norm.FuncName(), SumAbsFunc, 1)
+	tensor.AddFunc(Prod.FuncName(), ProdFunc, 1)
+	tensor.AddFunc(Min.FuncName(), MinFunc, 1)
+	tensor.AddFunc(Max.FuncName(), MaxFunc, 1)
+	tensor.AddFunc(MinAbs.FuncName(), MinAbsFunc, 1)
+	tensor.AddFunc(MaxAbs.FuncName(), MaxAbsFunc, 1)
+	tensor.AddFunc(Mean.FuncName(), MeanFunc, 1)
+	tensor.AddFunc(Var.FuncName(), VarFunc, 1)
+	tensor.AddFunc(Std.FuncName(), StdFunc, 1)
+	tensor.AddFunc(Sem.FuncName(), SemFunc, 1)
+	tensor.AddFunc(SumSq.FuncName(), SumSqFunc, 1)
+	tensor.AddFunc(L2Norm.FuncName(), L2NormFunc, 1)
+	tensor.AddFunc(VarPop.FuncName(), VarPopFunc, 1)
+	tensor.AddFunc(StdPop.FuncName(), StdPopFunc, 1)
+	tensor.AddFunc(SemPop.FuncName(), SemPopFunc, 1)
+	tensor.AddFunc(Median.FuncName(), MedianFunc, 1)
+	tensor.AddFunc(Q1.FuncName(), Q1Func, 1)
+	tensor.AddFunc(Q3.FuncName(), Q3Func, 1)
+}
+
 // Stats is a list of different standard aggregation functions, which can be used
 // to choose an aggregation function
 type Stats int32 //enums:enum
 
 const (
-	// count of number of elements
+	// count of number of elements.
 	Count Stats = iota
```

Comment:
If you're going to add the trailing period, which is good, you should also add the identifier name at the start.